blueocean-dashboard/src/main/js/components/MultiBranch.jsx

```diff
@@ -28,6 +29,7 @@ export class MultiBranch extends Component {
             this.pager = this.context.pipelineService.branchPager(organization, pipeline);
         }
         this._countExtensions();
+        analytics.trackPipelineBranches();
```

Comment:
@kzantow is `componentWillMount` the right place to send events, if the intent is to send the event every time the branch tab is clicked?

Reply:
I was looking at this, I would probably push all this stuff into the router instead of the components. `componentWillMount` will work for what you want, though, yes.

Reply:
Moved everything to the router level. Much cleaner.